### PR TITLE
poleval cyberbullying

### PR DESCRIPTION
**Pull request by czabo**

### CODE REVIEW

File: datasets/poleval2019_cyberbullying/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
+annotations_creators:
+- found
+language_creators:
+- found
+languages:
+- pl
+licenses: []
```
Comment (yjernite) [inline datasets/poleval2019_cyberbullying/README.md:7]:
```suggestion
licenses:
- unknown
```

**Comment by ijaz314** [comment]
Hii please follow me

**Review by lhoestq — APPROVED** [review]
Thank you !

File: datasets/poleval2019_cyberbullying/README.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+### Dataset Curators
+
+[More Information Needed]
+
+### Licensing Information
+
+[More Information Needed]
+
+### Citation Information
```
Comment (lhoestq) [inline datasets/poleval2019_cyberbullying/README.md:147]:
```suggestion
### Citation Information

```
@proceedings{ogr:kob:19:poleval,
  editor    = {Maciej Ogrodniczuk and Łukasz Kobyliński},
  title     = {{Proceedings of the PolEval 2019 Workshop}},
  year      = {2019},
  address   = {Warsaw, Poland},
  publisher = {Institute of Computer Science, Polish Academy of Sciences},
  url       = {http://2019.poleval.pl/files/poleval2019.pdf},
  isbn      = "978-83-63159-28-3"}
}
```
```

**Comment by lhoestq** [comment]
merging since the CI is fixed on master

**Pull request merged**